eslint.config.mjs

```diff
@@ -104,6 +106,19 @@ export default [
           argsIgnorePattern: "^_",
         },
       ],
+      "no-restricted-properties": [
```

Comment:
De rule is "native" eslint, hier is de plugin `eslint-plugin-cypress` niet voor benodigd. Dus die heb ik in verband met het releasen, er weer uitgehaald, maar deze mooie regel laat ik staan.

Reply:
Mocht blijken dat deze plugin op een andere manier van toegevoegde waarde was kunnen we die natuurlijk toevoegen